Examples/Run/MaterialMapping/GeantinoRecording/GeantinoRecordingExample.cpp

```diff
@@ -16,10 +16,10 @@
 #include "ACTFW/Framework/Sequencer.hpp"
 #include "ACTFW/Io/Root/RootMaterialTrackWriter.hpp"
 #include "ACTFW/Io/Root/RootSimHitWriter.hpp"
-#include "ACTFW/MaterialMapping/GeantinoRecording.hpp"
 #include "ACTFW/Options/CommonOptions.hpp"
-#include "ACTFW/Plugins/DD4hepG4/DD4hepToG4Svc.hpp"
 #include "ACTFW/Utilities/Paths.hpp"
+#include "ActsExamples/Geant4/DD4hepToG4Svc.hpp"
+#include "ActsExamples/Geant4/GeantinoRecording.hpp"
```

Comment:
Is it right to change some of these paths, but not all of them?

Reply:
Here, I only changed the paths that I had to move anyways. I am not sure it is right, but you have to start somewhere and I wanted to avoid bloating this PR with unrelated changes. I expect that we will have mix of conventions for a while until everything is converted.

Reply:
Yeah I think I agree, I'd have preferred the renaming to not be in this PR.

Reply:
Ok, I'll switch it back to the original names.

Reply:
The Geant4 consolidation work has been removed. I'll put that into a separate PR.